redmine_import_time_entry/unit/backend_adapter.py

```diff
@@ -0,0 +1,103 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 - Present Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.tools.translate import _
+from openerp.addons.connector.exception import (
+    NothingToDoJob, InvalidDataError)
+from redmine import exceptions
+from openerp.addons.connector_redmine.unit.backend_adapter import \
+    RedmineAdapter
+from openerp.addons.connector_redmine.backend import redmine13
+
+
+@redmine13
```

Comment:
should be registered on `@redmine` so if `@redmine14` (or whatever next release) is added, this code will still be available.